Frontend/ui-common/src/Engineering/Helpers/Utils.purs

```diff
@@ -122,3 +122,10 @@ getAppConfigImpl =
       Left err -> do
         _ <- pure $ printLog ("Not able to decode config" <> show err) config
         pure $ DefaultConfig.config
+
+pow :: Number -> Int -> Number
+pow _ 0 = 1.0
+pow base exponent = base * pow base (exponent - 1)
+
+truncate :: Int -> Number -> Number
+truncate precision num = (round (num * (10.0 `pow` precision))) / (10.0 `pow` precision)
```

Comment:
we already have a function called `parseFloat` which takes the number and precision, please check that

Reply:
done, using parseFloat function for showing instead of saving truncated value in rating field